api/types/plugins/logdriver/entry.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
```

Comment:
It would be nice to move the existing `api/types/plugins_*` code under `api/types/plugins` dir, now that we have one.